src/zenml/cli/secret.py

```diff
@@ -47,7 +47,7 @@ def validate_kv_pairs(key: Optional[str], value: Optional[str]) -> bool:
 @cli.group(cls=TagGroup, tag=CliCategories.IDENTITY_AND_SECURITY)
 @click.pass_context
 def secret(ctx: click.Context) -> None:
-    """Secrets for storing key-value pairs for use in authentication."""
+    """List and manage Secrets of key-value pairs."""
     # TODO [ENG-724]: Ensure the stack actually contains an active secrets
     #  manager
```

Comment:
Is this necessary any more?

Reply:
yes removed that and another one that was already done, I think we should give our list of TODO a visit. I'm assuming there is a good part of them already done or no need to be but not removed from the code.

Reply:
You mean inside JIRA? If I'm not mistaken, when you remove them from the codebase, they automatically will also get removed as tickets from JIRA.

Reply:
If you mean inside the code, then agreed. Would make for a nice small PR as well :)

Reply:
yes for code, that is what I assumed as well removing from them the codebase is enough. will create it then :)